extension/indexRepo.js

```
@@ -40,14 +55,88 @@ export async function buildIndex(rootDir, apiKey) {
+            } else if (
+              callee.type === "MemberExpression" &&
+              callee.property.type === "Identifier"
+            ) {
+              calleeName = callee.property.name;
+            }
+            if (!calleeName) return;
+            const func = ancestors
+              .slice()
+              .reverse()
+              .find((a) => a.type === "FunctionDeclaration" && a.id);
+            if (func && func.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)